.eslintrc.yml

```diff
@@ -645,3 +651,24 @@ rules:
     - error
   lodash/prefer-times:
     - error
+
+  # JSDoc
+
+  jsdoc/check-param-names:
+    - error
+  jsdoc/check-tag-names:
+    - error
+  jsdoc/newline-after-description:
+    - error
+  jsdoc/require-example:
```

Comment:
There's a few places in this PR where it adds `/* eslint-disable jsdoc/require-example */` ;-)